### PR TITLE
Update kfp dependency to correct version

### PR DESCRIPTION
**Pull request by akchinSTC**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by akchinSTC** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "kfp-tekton" %}
-{% set version = "1.1.1" %}
+{% set version = "1.1.0" %}
```
Comment (ckadner) [inline recipe/meta.yaml:2]: @akchinSTC -- if we are patching a previous version, do we have to then subsequently have re-update the latest version after merging this PR?

**Review by ckadner — COMMENTED** [review]
Hi @akchinSTC -- you may know better than I, but do you have a link describing the process for patching older versions of conda-forge recipes?

As far as I can find, we would need to create branches for several versions:
https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-several-versions

> If you’d like to maintain more than one version of your package, you can use branches on the feedstock. To do this, fork your feedstock and make a meaningful branch name (e.g., v1.X or v1.0). Make the required changes to the recipe and rerender the feedstock. Then push this branch from your fork to the upstream feedstock. Our CI services will automatically build any branches in addition to the default branch.

File: recipe/meta.yaml (hunk 1 above)
Comment (akchinSTC) [reply recipe/meta.yaml:2]: @ckadner - I just spoke to @Tomcli and after reading your link, we should probably open a support branches for 1.0.x and 1.1.x ? Ill rebase my PRs on those two and if we merge the PRs in the order (1.1.0 then 1.1.1) we should keep things spiffy and neat

**Pull request closed without merging**